bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/impl/SingleProbeStrategyParams.scala

```diff
@@ -40,7 +38,9 @@ sealed trait SingleProbeStrategyParams {
         case SingleProbeStrategyParams.AltairAowfsParams => true
         case _                                           => false
       }
-      CatalogQuery.coneSearch(base.toNewModel, rc, ml, catalogName, isLgs)
+      // for Altair two catalogs can be searched
+      val cats = if (isLgs) NonEmptyList(GaiaGemini, UCAC4) else catalogs
+      cats.map(c => CatalogQuery.coneSearch(base.toNewModel, rc, ml, c, isLgs))
```

Comment:
There is an `AltairAowfsParams` entry.  Perhaps we could override `catalogs` there to return `GaiaGemini` and `UCAC4`?  I suppose it doesn't make much difference though since it seems we are computing `isLgs` for some other reason.